test/workbox-google-analytics/integration/test-all.js

```diff
@@ -152,7 +159,9 @@ describe(`[workbox-google-analytics] initialize`, function() {
     // Ensure the hit's qt params were present and greater than 0,
     // and ensure those values reflect the original order of the hits.
     expect(requests[0].params.qt > 0).to.be.true;
-    expect(requests[0].params.qt > 0).to.be.true;
-    expect(requests[0].originalTime < requests[1].originalTime).to.be.true;
+    expect(requests[1].params.qt > 0).to.be.true;
+    // This should be <= instead of less than, since a ms didn't necessarily
+    // pass in between the two events.
+    expect(requests[0].originalTime <= requests[1].originalTime).to.be.true;
```

Comment:
I feel like this should be `<=` instead of `<`, because I'm really not sure how much time has elapsed in between the two GA events.

Reply:
Hmmm, but then I think the timestamp update logic could be removed and the test would still pass.

I'd rather sleep for 1ms and continue to assert `<` (though I'm pretty sure 1ms will also elapse since we're waiting for a postMessage to succeed -- but I get your point).

Reply:
I've added back the timeouts, so I'm fine with using `<` now (since actual time will definitely have passed).